app/models/invoice.rb

```diff
@@ -3,8 +3,8 @@ class Invoice < ApplicationRecord
 
   has_many :invoice_items
   has_many :items, through: :invoice_items
-  has_many :merchants, through: :items
+  # has_many :merchants, through: :items
```

Comment:
Sam and I decided that it wouldn't hurt to add this just in case, but now we understand having it does affect the rake tasks.